app/controllers/api/v1/admin/football_pitches.rb

```diff
@@ -0,0 +1,87 @@
+module API
+  module V1
+    module Admin
+      class FootballPitches < Grape::API
+        include API::V1::Helpers::PaginationHelper
+
+        before do
+          authenticate_user!
+          require_admin
+        end
+
+        helpers do
+          def load_football_pitch
+            @football_pitch = FootballPitch.find_by id: params[:id]
+
+            return if @football_pitch
+
+            error!({message: "Football pitch not found"}, 404)
+          end
+
+          def have_booking_pending?
+            @football_pitch.bookings.booking_status(:pending).empty?
```

Comment:
cái booking gọi kiểu enum dc k?
```suggestion
            @football_pitch.bookings.pending.empty?
```